api/Application.Infrastructure/CPManagement/CPManagementService.cs

```diff
@@ -84,13 +84,9 @@ public PagedList<CourseProjectData> GetProjects(int userId, GetPagedListParams p
             {
                 var courseProjects = from cp in query
                                      let acp = cp.AssignedCourseProjects.FirstOrDefault()
-                                     where (acp.Student.LastName.Contains(searchString) ||
+                                     where acp.Student.LastName.Contains(searchString) ||
                                             cp.Theme.Contains(searchString) ||
-                                            acp.Student.Group.Name.Contains(searchString)) &&
-                                            acp.Student.IsActive.HasValue &&
-                                            acp.Student.IsActive.Value &&
-                                            acp.Student.Confirmed.HasValue &&
-                                            acp.Student.Confirmed.Value
```

Comment:
I still don't understand why this should be removed?

Reply:
this file does not need to be changed, I accidentally uploaded it

Reply:
I still see the changes, roll them back